crossbeam-utils/src/cache_padded.rs

```diff
@@ -207,3 +207,9 @@ impl<T> From<T> for CachePadded<T> {
         Self::new(t)
     }
 }
+
+impl<T: fmt::Display> fmt::Display for CachePadded<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "Cache padding with length {}", &self.value)
```

Comment:
I would have expected it to work the same way as when called via deref... Like:
```suggestion
        fmt::Display::fmt(&self.value, f)
```

cc @akanalytics